pkg/accounts/router.go

```diff
@@ -132,33 +132,32 @@ func createCustomerAccount(logger log.Logger, repo Repository, fedClient fed.Cli
 			return
 		}
 		if err := request.validate(); err != nil {
-			logger.LogErrorF("problem validating account: %v", err)
 			moovhttp.Problem(w, err)
 			return
 		}
 		if err := request.disfigure(keeper); err != nil {
-			logger.LogErrorF("problem disfiguring account: %v", err)
+			logger.LogErrorf("problem disfiguring account: %v", err)
```

Comment:
From the link you shared it says:
> Do not wrap an error when doing so would expose implementation details.
I don't understand how we may benefit here from wrapping the error. `moovhttp.Problem` has no context about the received error and will not be able to benefit from unwrapped error.

Would love to learn how it may be useful.